### PR TITLE
Add migration docs

### PR DESCRIPTION
**Pull request by ChaoLiangSuper**
Adding migration nots to the doc
including:
- Popover
- Checkbox
- Tabs&Tab
- Accordion

### CODE REVIEW

File: docs/components/MigrationNote/index.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    this.setState(prev => ({ showModal: !prev.showModal }));
+  };
+
+  render() {
+    return (
+      <>
+        <Button bsStyle="link" onClick={this.openCollapse} className="migration-docs">
+          <b>Version 27 Migration guide </b>
+          <span role="img" aria-label="emoji">
+            📚
```
Comment (lteacher) [inline docs/components/MigrationNote/index.jsx]: What emoji is this? lol

File: docs/components/MigrationNote/index.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        </Button>
+        <Collapse in={this.state.showModal} mountOnEnter>
+          <div>
+            <h2>Migration Guide</h2>
+            <hr />
+            <h3>Popover Component</h3>
+            <p>
+              <b>
+                {`<Popover />`} component has accepted theme. The theme prop will accept{' '}
+                {`['light', 'dark', 'warn', 'error']`}.
```
Comment (lteacher) [inline docs/components/MigrationNote/index.jsx]: Not sure the wording of above. Is it supposed to be something like

>_The \<Popover /\> component can be themed by providing one of the following values to the `theme` prop:_

File: docs/components/MigrationNote/index.jsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+              </b>
+              <br />
+              <b>
+                For more information check the example: <a href="#popover-example">Popover Example</a>
+              </b>
+            </p>
+            <br />
+            <h3>Accordion Component</h3>
+            <p>
+              <b>The accordion component has been rewritten due to a performance issue.</b>
```
Comment (lteacher) [inline docs/components/MigrationNote/index.jsx]: > The accordion component has been rewritten due to performance issues.

or

> The accordion component has been rewritten due to performance concerns.

File: docs/components/MigrationNote/index.jsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+              <br />
+              <b>The new component will not accept {`'panels'`} in the props.</b>
```
Comment (lteacher) [inline docs/components/MigrationNote/index.jsx]: > The new component no longer accepts the `panels` prop. Instead, .....

Something like assuming:
> ... each panel should be provided as a child of the Accordion component

File: docs/components/MigrationNote/index.jsx (hunk 1 above)
Comment (ChaoLiangSuper) [reply docs/components/MigrationNote/index.jsx]: I think it means for 'documents' 'books' HAHA

File: docs/components/MigrationNote/index.jsx [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+          <div>
+            <h2>Migration Guide</h2>
+            <hr />
+            <h3>Popover Component</h3>
+            <p>
+              <b>
+                {`<Popover />`} component has accepted theme. The theme prop will accept{' '}
+                {`['light', 'dark', 'warn', 'error']`}.
+              </b>
+              <br />
```
[the remark this answers is not in the document]
Comment (ChaoLiangSuper) [reply docs/components/MigrationNote/index.jsx]: Hmm good point, I will change it.

**Pull request merged**